doc/languages-frameworks/rust.section.md

```diff
@@ -127,7 +129,7 @@ where they are known to differ. But there are ways to customize the argument:
    }
    ```
    will result in:
-   ```shell
+   ```ShellSesssion
```

Comment:
Not sure on those examples.

Reply:
Have you tried building the manual to see if it renders these correctly? I've not seen this sort of meta language before.

Reply:
These are not shell sessions (there are no prompts). I would use `shell` at most but perhaps it can do without a language.

Reply:
> Have you tried building the manual to see if it renders these correctly?

No but my vim highlights it. :P

and shell looks better indeed.